snippets/base/forms.py

```diff
@@ -119,6 +119,24 @@ class BaseSnippetAdminForm(forms.ModelForm):
 class SnippetAdminForm(BaseSnippetAdminForm):
     template = forms.ModelChoiceField(queryset=SnippetTemplate.objects.exclude(hidden=True),
                                       widget=TemplateSelect)
+    client_option_has_fxaccount = forms.ChoiceField(
+        label='Firefox Account',
+        choices=(('any', 'Show to all users'),
+                 ('yes', 'Show only to users with an enabled Firefox Account'),
+                 ('no', 'Show only to users without an enabled Firefox Account')))
+    client_option_has_testpilot = forms.ChoiceField(
+        label='Test Pilot',
+        choices=(('any', 'Show to all users'),
+                 ('yes', 'Show only to users with TestPilot'),
+                 ('no', 'Show only to users without TestPilot')))
```

Comment:
I am just commenting here because these fields are required but you are allowing a default empty value when saving an object.  If this is intended, please ignore